api/services/Frontend.js

```diff
@@ -67,15 +67,13 @@ module.exports = {
 
     root: () => url('/app'),
 
-    comment: function (comment, group) {
-      // TODO: update to use comment specific url when implemented in frontend
-      let groupSlug = getSlug(group)
+    comment: function ({ comment, groupSlug, post }) {
 
       let groupUrl = isEmpty(groupSlug) ? '/all' : `/groups/${groupSlug}`
 
-      const postId = comment.relations.post.id
+      const postId = comment?.relations?.post?.id || post.id
```

Comment:
why would we pass the post in? cant we always get it from the comment? seems dangerous you could pass in a different post than the one the comment is on

Reply:
Previously there was only one call site for this function, now three. One of the new call-sites was passing in a comment model that didn't have a post on it. So decided to pass in the post in that instance.